plaid/src/apis/npm/npm_web_client.rs

```diff
@@ -107,35 +108,43 @@ impl Npm {
             ])
             .send()
             .await
+            .map(|_| ())
```

Comment:
I assume that this means there is no value in the response which could be useful to the caller. If so, just resolve this comment.